system/modules/core/templates/modules/mod_breadcrumb.html5

```diff
@@ -2,14 +2,14 @@
 
 <?php $this->block('content'); ?>
 
-  <ul>
+  <ol itemscope itemtype="http://schema.org/BreadcrumbList">
```

Comment:
This is wrong (in terms of backwards compatibility). The list must remain unordered.